### DISCUSSION TITLE
Date recorded has 9 hours time difference

### DISCUSSION THREAD

**Discussion by Klagio** [teslamate-org/teslamate#1240] [Q&A]
How to use local time (GMT +1, Central Europe), my data has 9 hours jetlag

**Describe the bug**

**Expected behavior**

**How to reproduce it (as minimally and precisely as possible)**:

**Relevant entries from the logs**

```
paste logs here
```

**Screenshots**

**Data**

**Environment**

- TeslaMate version:
- Type of installation: Docker or Manual
- OS TeslaMate is installed on:
- User OS & Browser:
- Others:

**Comment by clausbruun** [comment]
Set `TM_TZ=Europe/Copenhagen` in your **.env** file.

**Comment by Klagio** [comment]
Thanks, but where is this file?

**Comment by clausbruun** [comment]
I used this guide: https://docs.teslamate.org/docs/guides/traefik#env

**Comment by Klagio** [comment]
Thanks but I have a Caddy 2 setup.

Perhaps I just need to create a.    
.env
file under /home/pi   where the file docker-compose is??

**Comment by clausbruun** [comment]
> file under /home/pi where the file docker-compose is??

Yes, in the home directory.

**Comment by Klagio** [comment]
Or I just put


      - TZ=Europe/Rome 
      -
In the docket compose under Teslamate?

**Comment by DrMichael** [comment]
Or rather
`      - PGTZ=Europe/Rome
in the database section? Wit that grafana connects to postgresql with that timezone.
